lib/ht_sip_validator/validation/image/count.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module HathiTrust::Validation::Image
+  class Count < HathiTrust::Validation::Base
+
+    def perform_validation
+      image_files = HathiTrust::Validation::Image.image_files(@sip.files)
+      
+      if image_files.count != text_files.count
```

Comment:
We probably won't want to enable this validator by default - while some institutions might want to check this is the case, in general, it's OK to be missing OCR for some images. (It would NOT be OK to have OCR with no corresponding image, though!)